### DISCUSSION TITLE
Magic link - expired token issue

### DISCUSSION THREAD

**Discussion by nik-orda** [outline/outline#8051] [Self Hosting] [answered]
Hi!

I'm testing self-hosted installation of the Outline wiki in Docker and there is an issue I'm facing:

When I'm trying to use magic link, I'm getting "the token is expired" issue. It happens only with my copr mail address in Google Workspace like po@mycompany.com or bot@mycompany.com. I try to login, then I receive an email in a few seconds, and the follow the link - it says that the token is expired. It took only a few seconds. How can it be expired?

When I use my personal Google mail address - it works fine and I can login.

The configuration is pretty basic, I have only changed the domain and connected to the company's Google SAML auth.

What can be wrong here?

**Comment by tommoor** [comment, marked as answer]
It is either expired because the time on your server has drifted further than 10 minutes from realtime (very unlikely), or because the link was already followed (tokens are single use).

It seems the later is most likely here, some sort of anti-virus software is accessing the link and "spending" it before you are able to click. We have various mechanism in place to prevent this, but only so much that can be done.

**Comment by nik-orda** [comment]
> It seems the later is most likely here, some sort of anti-virus software is accessing the link and "spending" it before you are able to click. 

It looks like that's the case. Thank you!
